modifiers/heading_id.test.js

```diff
@@ -122,3 +122,12 @@ it('supports a custom sluggify function', () => {
     expect(mdSluggify.render('# Hello World!')).toBe(`<h1 id="helloworld">Hello World!</h1>
 `);
 });
+
+const mdHashLink = require('markdown-it')().use(require('./heading_id'), {
+    hashLink: true,
+});
+
+it('supports generating a hashlink', () => {
+    expect(mdHashLink.render('# Hello World!')).toBe(`<h1 id="hello-world"><a class="hash-anchor" href="#hello-world" aria-hidden="true"></a>Hello World!</h1>
+`);
+});
```

Comment:
Can we add a test to cover the level restriction (and a test for the class name once that setting is added)?